### PR TITLE
Remove sov chain_state

### PR DESCRIPTION
**Pull request by kpp**
# Description
Remove modules `sov-chain-state`, `sov-blob-storage`, `sov-attester-incentives`, `sov-soft-confirmations-kernel` and chain_state.json files

## Linked Issues
- Fixes https://github.com/chainwayxyz/citrea/issues/513

## Testing
`make test`

### CODE REVIEW

File: crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs [hunk 1]
```diff
@@ -447,9 +447,9 @@ where
     ) -> (Self::StateRoot, Self::ChangeSet) {
         let mut working_set = StateCheckpoint::new(pre_state.clone()).to_revertable();
 
-        self.kernel
-            .genesis(&params.kernel, &mut working_set)
-            .expect("Kernel initialization must succeed");
+        // self.kernel
+        //     .genesis(&params.kernel, &mut working_set)
+        //     .expect("Kernel initialization must succeed");
```
Comment (eyusufatik) [inline crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs:452]: you should also remove kernel from the stf blueprint

File: crates/sovereign-sdk/examples/demo-stf/src/tests/mod.rs
Comment (eyusufatik) [inline crates/sovereign-sdk/examples/demo-stf/src/tests/mod.rs:1]: why?

File: crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs [hunk 2]
```diff
@@ -497,51 +477,7 @@ where
...
-            selected_blobs.len()
-        );
-
-        let mut checkpoint = batch_workspace.checkpoint();
-
-        let mut batch_receipts = vec![];
-
-        for (blob_idx, mut blob) in selected_blobs.into_iter().enumerate() {
-            let (apply_blob_result, checkpoint_after_blob) =
-                self.apply_blob(checkpoint, blob.as_mut_ref());
```
Comment (eyusufatik) [inline crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs:520]: can use blobs here in place for selectted blobs, i assume this breaks sov-sdk tests

File: crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/stf_blueprint.rs [hunk 3]
```diff
@@ -277,222 +230,6 @@ where
             (Err(err), batch_workspace) => (Err(err), batch_workspace.revert()),
         }
     }
-    #[cfg_attr(all(target_os = "zkvm", feature = "bench"), cycle_tracker)]
```
Comment (eyusufatik) [inline crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/stf_blueprint.rs:280]: I am really surprised this didn't break anything :D

File: crates/sovereign-sdk/examples/demo-stf/src/tests/mod.rs
Comment (kpp) [reply crates/sovereign-sdk/examples/demo-stf/src/tests/mod.rs:1]: We don't need these tests

**Review by eyusufatik — APPROVED** [review]

**Pull request merged**